upgrade/upgrade.go

```diff
@@ -286,6 +286,10 @@ func doResourceUpgrade(namespace string, lhClient *lhclientset.Clientset, kubeCl
 		return err
 	}
 
+	if err := upgradeutil.CopyCSISidecarSettings(namespace, lhClient); err != nil {
+		return err
+	}
+
```

Comment:
Hello @semenas,
We should add `v110xto1110` in https://github.com/longhorn/longhorn-manager/tree/master/upgrade instead and put the migration codes in the folder.

For upgrade/upgrade.go, we need to add 
```
	if semver.Compare(lhVersionBeforeUpgrade, "v1.11.0") < 0 {Add a comment on  line R255Add diff commentMarkdown input:  edit mode selected.WritePreviewAdd a suggestionHeadingBoldItalicQuoteCodeLinkUnordered listNumbered listTask listMentionReferenceSaved repliesAdd FilesPaste, drop, or click to add filesCancelCommentStart a reviewReturn to code
		logrus.Info("Walking through the resource upgrade path v1.10.x to v1.11.0")
		if err := v110xto1110.UpgradeResources(namespace, lhClient, kubeClient, resourceMaps); err != nil {
			return err
		}
	}
```